python/modem-firmware-1.3+/device_credentials_installer.py

```diff
@@ -528,7 +569,8 @@ def main():
         attr = ',\"CN={}\"'.format(custom_dev_id)
 
     write_line('AT%KEYGEN={},2,0{}'.format(args.sectag,attr))
-    retval, output = wait_for_prompt(b'OK', b'ERROR', store=b'%KEYGEN:')
+    # include the CRLF in OK because 'OK' could be found in the CSR string
```

Comment:
good catch

Reply:
took me way too long to figure out this is why KEYGEN occasionally failed 🥴